<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>1066</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1500</width>
    <height>1066</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gestion des étudiants</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../etudiant.png</normaloff>../../etudiant.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>1000</width>
     <height>1000</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>2000</width>
     <height>2000</height>
    </size>
   </property>
   <widget class="QTableWidget" name="tableWidgetEtudiant">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>461</width>
      <height>541</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::DoubleClicked</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>531</width>
      <height>541</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QLineEdit" name="lineEditNom">
     <property name="geometry">
      <rect>
       <x>252</x>
       <y>70</y>
       <width>151</width>
       <height>33</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditPrenom">
     <property name="geometry">
      <rect>
       <x>242</x>
       <y>160</y>
       <width>171</width>
       <height>33</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxOption">
     <property name="geometry">
      <rect>
       <x>262</x>
       <y>290</y>
       <width>151</width>
       <height>29</height>
      </rect>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonAbandonner">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>390</y>
       <width>95</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Abandonner</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSupprimer">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>390</y>
       <width>95</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonModifier">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>390</y>
       <width>95</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonAjouter">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>390</y>
       <width>95</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>66</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>170</y>
       <width>66</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Prénom</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>290</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Option</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>fichier</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionQuiter"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuiter">
   <property name="text">
    <string>&amp;quitter</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tableWidgetEtudiant</tabstop>
  <tabstop>lineEditNom</tabstop>
  <tabstop>lineEditPrenom</tabstop>
  <tabstop>comboBoxOption</tabstop>
  <tabstop>pushButtonAjouter</tabstop>
  <tabstop>pushButtonModifier</tabstop>
  <tabstop>pushButtonSupprimer</tabstop>
  <tabstop>pushButtonAbandonner</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
